### ISSUE TITLE
npm outdated -g doesn't work

### ISSUE THREAD

**Issue by welljj** [coreybutler/nvm-windows#902]
**Issue:**
`npm outdated -g` doesn't return anything.

**How To Reproduce:**
1. Fresh install nvm-windows
2. Install an older version of any package. Ex: `npm install -g npm-check-updates@16.5.1`
3. Run `npm outdated -g`

**Expected Behavior:**
![image](https://user-images.githubusercontent.com/32354652/207966487-302aa1e9-b2c6-4616-9d04-e7e85e7a3ad0.png)
I should return a list of outdated global packages, like the attached photo

**Desktop:**
OS: Windows 11 22H2

**Comment by coreybutler** [comment]
This is an npm issue, not an NVM4W issue. NVM4W doesn't touch npm.

I looked into the npm code and I'm pretty sure the problem is at https://github.com/npm/cli/blob/latest/lib/commands/outdated.js#L144. The code is 

```js
for (const child of node.children.values()) {
  //...
}
```

The problem is `node.children` doesn't exist, but `node.target.children` does. So, `npm outdated` seems to think there are no global modules.

I injected the following code into npm's outdated.js file on between lines 143 and 144:

```js
console.log({ children: node.children, target: node.target.children })
```

The results in a list of the global modules.

<img width="631" alt="image" src="https://user-images.githubusercontent.com/770982/207986816-aaaf9cdf-fd62-4f23-aa26-1221c35276cc.png">

Bottom line, this is an npm issue, not an NVM4W issue. Feel free to use this to file an issue over at https://github.com/npm/cli/issues.
